src/tensors/cpu/wasm_intgemm_fallback.cpp

```diff
@@ -64,11 +64,19 @@ extern "C" void int8PrepareBiasFallback(const int8_t* input_B_prepared,
                                         const float* input_bias,
                                         float* output) {
   float unquant_factor = (-1) * ((127.0f / scale_A) * (127.0f / scale_B)) / (127.0f);
-  intgemm::Int8Shift::PrepareBias(
-      input_B_prepared,
-      width,
-      cols_B,
-      intgemm::callbacks::UnquantizeAndAddBiasAndWrite(unquant_factor, input_bias, output));
+  if (input_bias == nullptr) {
```

Comment:
Minor, stylistic. I prefer `if (input_bias)`, and I have used this style of `nullptr` checking elsewhere.